apps/typegpu-docs/src/content/examples/tests/tests/index.ts

```diff
@@ -0,0 +1,48 @@
+import tgpu from 'typegpu';
+import * as d from 'typegpu/data';
+import { booleanTests } from './boolean';
+
+const root = await tgpu.init();
+
+const resultBuffer = root.createBuffer(d.i32, 0).$usage('storage');
+const result = resultBuffer.as('mutable');
```

Comment:
We can use the unstable `createMutable` API to create `result`, as it's only used as mutable storage.
```suggestion
const root = await tgpu.init();
const result = root['~unstable'].createMutable(d.i32, 0);
```